README.md

```diff
@@ -76,10 +76,25 @@ config.vm.synced_folder ".", "/home/core/share", id: "core", :nfs => true,  :mou
 
 After a 'vagrant reload' you will be prompted for your local machine password.
 
-#### Provisioning with user-data
+#### Provisioning with Ignition (VirtualBox Provider (default))
 
-The Vagrantfile will provision your CoreOS VM(s) with [coreos-cloudinit][coreos-cloudinit] if a `user-data` file is found in the project directory.
-coreos-cloudinit simplifies the provisioning process through the use of a script or cloud-config document.
+When using the VirtualBox provider for vagrant (the default), Ignition is used to provision the machine. This uses a special plugin that is
+automatically installed when using the default Vagrantfile. The config options for the plugin are all prefixed with `config.ignition` and can
+be found in this Vagrantfile or in the README of the [plugin][https://github.com/coreos/vagrant-ignition].
+
+To get started, copy `config.ign.sample` to `config.ign` and make any necessary modifications.
+Check out the [Ignition Getting Started documentation][ignition] to learn about the available featrues.
+
+[ignition]: https://github.com/coreos/ignition/blob/master/doc/getting-started.md
+
+NOTE: The provided `config.ign.sample` contains a few variables (`{{HOSTNAME}}`, `{{PRIVATE_IP}}`, `{{CLUSTER}}`) that the Vagrantfile uses
+to configure etcd clusters and are not part of the Ignition spec.
+
+#### Provisioning with user-data (VMWare provider)
+
+When using the VMWare provider for vagrant, the Vagrantfile will provision your CoreOS VM(s)
```

Comment:
This is okay for now, but we should also add support for VMware to Vagrant (Ignition already knows how to talk to VMware).